dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/LogsConnector.java

```diff
@@ -16,19 +16,21 @@
  */
 package com.google.edwmigration.dumper.application.dumper.connector;
 
-import java.text.Format;
-import java.text.SimpleDateFormat;
+import com.google.edwmigration.dumper.application.dumper.utils.ArchiveNameUtil;
+import java.time.Clock;
 import javax.annotation.Nonnull;
-import org.apache.commons.lang3.StringUtils;
 
 /** @author shevek */
 public interface LogsConnector extends Connector {
 
   @Nonnull
+  @Override
   default String getDefaultFileName(boolean isAssessment) {
-    Format format = new SimpleDateFormat("yyyyMMdd'T'HHmmss");
-    String timeSuffix = "-" + format.format(System.currentTimeMillis());
-    return String.format(
-        "dwh-migration-%s-logs%s.zip", getName(), isAssessment ? timeSuffix : StringUtils.EMPTY);
+    if (isAssessment) {
+      Clock clock = Clock.systemDefaultZone();
+      return ArchiveNameUtil.getFileNameWithTimestamp(getName(), "logs", clock);
+    } else {
+      return ArchiveNameUtil.getFileName(getName(), "logs");
+    }
```

Comment:
```suggestion
    }
    return ArchiveNameUtil.getFileName(getName(), "logs");
```

Reply:
Done

Reply:
looks like `else` statement is still there. It is recommended to skip it alter the `return` statement. Could you please remove it?